src/ngraph/op/experimental/quantized_conv.cpp

```diff
@@ -40,15 +40,17 @@ op::QuantizedConvolution::QuantizedConvolution(const shared_ptr<Node>& data_batc
     , m_data_dilation_strides(data_dilation_strides)
     , m_requantize(requantize)
 {
-    constructor_validate_and_infer_types();
-
     //TODO(nbpatel): Add checks.
```

Comment:
This should all be done in `validate_and_infer_types`, which is when element type and shape propagation will happen in the near future. Likewise, you need to be calling `constructor_validate_and_infer_types` which will become a no-op and then be removed in the near future, but, currently, does the type/shape propagation.